Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -795,6 +795,12 @@ def audit_specs
         return if stable_url_minor_version.even?
 
         problem "#{stable.version} is a development release"
+      when %r{^https://github.com/}
+        _, owner = URI.parse(stable.url).path.split("/")
+        tag = stable.specs[:tag] || stable.version.to_s
+        if GitHub.open_api("#{GitHub::API_URL}/repos/#{owner}/#{stable.full_name}/releases/tags/#{tag}")["prerelease"]
```

Comment:
Need to handle if this URL isn't found. You can reproduce this locally by editing the URL and ensuring it doesn't fail with a known-bad URL.